docusaurus.config.js

```diff
@@ -196,8 +194,11 @@ using OpenFga.Sdk.Configuration;`,
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
       metadata: [
-        { name: 'keywords', content: 'OpenFGA, open source, fine-grained-authorization, fine grained authorization, Zanzibar' },
-        { property: 'og:image', content: 'https://openfga.dev/img/openfga_logo.svg'  }
+        {
+          name: 'keywords',
+          content: 'OpenFGA, open source, fine-grained-authorization, fine grained authorization, Zanzibar',
+        },
+        { property: 'og:image', content: 'https://openfga.dev/img/og-rich-embed.png' },
```

Comment:
I don't think this absolute path actually resolves. Does anyone know how Docusaurus handles the `@static` folder? Do I have to include this image as an import (and can I do that in the config.js file?)?

Reply:
Anything in static folder is public.  For example, https://openfga.dev/img/chevron.svg is https://github.com/openfga/openfga.dev/blob/main/static/img/chevron.svg

Reply:
I am able to see http://localhost:3000/img/og-rich-embed.png

<img width="2032" alt="Screen Shot 2022-06-14 at 3 51 50 PM" src="https://user-images.githubusercontent.com/10730463/173676627-095702ff-77c7-4322-af3d-b17a9adf7613.png">

Reply:
Yeah, should be able use relative path where the root is the static folder.

ex:

![Screen Shot 2022-06-15 at 9 33 36 AM](https://user-images.githubusercontent.com/3803431/173840112-c62ae34a-2976-44d1-ba84-72badf6fe6ed.png)